zenhappens/src/styles/elements/_landinginfo.scss

```diff
@@ -0,0 +1,34 @@
+.landing__info{
+    padding: $padding-mobile;
+
+    @media screen and (min-width: 450px) {
+        padding: 2em;
+    }
+
+    @media screen and (min-width: 700px) {
+        display: flex;
+        flex-direction: row-reverse;
+        justify-content: space-between;
+
+        > section{
+            flex-basis: 40%;
+            // flex: 1 0 30%;
+        }
+    }
+
+    @media screen and (min-width: $display-burger) {
+        padding: 2em;
+    }
+}
+
+.hours, .philosophy{
+    margin: 1.8rem auto 1.8rem auto;
+
+    @media screen and (min-width: 700px) {
+        margin: 2em auto 2em auto;
```

Comment:
Same note as before about not duplicating measurements and making it explicit what changes in the media query.